src/services/GridConfig.js

```diff
@@ -44,7 +44,7 @@ export default new Vue({
 	methods: {
 		handleWindowResize() {
 			// find the first grid size that fit the current window width
-			const currentSize = Object.keys(sizes).find(size => size > document.documentElement.clientWidth)
+			const currentSize = Object.keys(sizes).find(size => size > document.documentElement.clientWidth || size === 'max')
 			this.gridConfig = sizes[currentSize]
```

Comment:
I would prefer the fallback on the this.gridConfig assignment instead.
```suggestion
			// fallback to max if none matches
			this.gridConfig = sizes[currentSize] || sizes['max']
```

Reply:
This seems to be more elegant, sorry I am not a javascript guy 😊😊

Reply:
No worries :)

Can you do the change so I can approve? :)